core/lib/libtomcrypt/sub.mk

```diff
@@ -114,6 +114,7 @@ _CFG_CRYPTO_WITH_ASN1 := $(call cryp-one-enabled, RSA DSA ECC)
 _CFG_CRYPTO_WITH_FORTUNA_PRNG := $(call cryp-all-enabled, AES SHA256)
 
 cppflags-lib-$(CFG_CRYPTO_SIZE_OPTIMIZATION) += -DLTC_SMALL_CODE -DLTC_NO_FAST
+cppflags-lib-$(CFG_CRYPTO_SIZE_OPTIMIZATION) += -Os
```

Comment:
In addition to Jens' comment (should be `cflags-`), I am wondering if `-Os` really belongs here. OK with setting `-DLTC_SMALL_CODE`, but for `-Os`, we already have a global setting, `platform-cflags-optimization` which is either `-O0` (debug) or `-Os` (not debug).

As for `-DLTC_NO_FAST`: it makes almost no difference as far as the size of libtomcrypt.a is concerned. I think it should not be set here (also I saw no performance difference on HiKey, 64-bit).